### ISSUE TITLE
macos play media key stopped working

### ISSUE THREAD

**Issue by dmlemos** [mpv-player/mpv#7446]
### Important Information

Provide following Information:
- mpv version: 0.32.0
- macOS Version: 10.14.6
- Source of the mpv binary or bundle: https://github.com/mpv-player/mpv/archive/v0.32.0.tar.gz
- If known which version of mpv introduced the problem: 0.32.0

### Reproduction steps

1. Play an audio file
2. Press the 'Play' media key (F8)

Starting having this issue from v0.32.0

### Expected behavior

Playback should be paused

### Actual behavior

Music continues

### Log file

https://0x0.st/iiOW.txt

**Comment by Akemi** [comment]
sry, but you are clearly not telling me the right source of the binary/bundle. the log is clearly from stolendata's builds and not from that source. stolendata's bundles are build without media key support because he explicitly disables them. see your log and `--disable-macos-media-player`.

**Comment by dmlemos** [comment]
Ohh... I was mistaken myself! I had the mpv cask installed with `brew cask install mpv`, which comes from `https://laboratory.stolendata.net/~djinn/mpv_osx/`.
I actually checked the previous formulas in git to see what had changed, but didn't see anything that changed besides version. Problem is I was checking the wrong formula (not the cask).

Works fine with `brew install mpv`, which installs from the source mentioned.
At some point I've moved to the cask, but can't remember the reason.

Thanks for pointing that out, made me see the mistake :)
